website/docs/components/pagination/index.js

```diff
@@ -413,7 +413,7 @@ export default class Index extends Component {
   get demoPaginatedDataCompact() {
     const { direction, cursorIndex } = getCursorParts(
       this.demoCurrentCursor,
-      this.model.records
+      this.model.records,
```

Comment:
these are arguments, not a list (essentially I _think_  we're passing three arguments instead of two, right?)